src/make_data/banner_intervention/email_selects_one_result.sql

```diff
@@ -0,0 +1,43 @@
+/* This query extracts the total number and proportion of sessions that select the
+   checker via email that selects at least one link on the results page
+*/
+
+
+DECLARE start_date STRING DEFAULT "20210803";
+DECLARE end_date STRING DEFAULT "20210816";
+
+CREATE OR REPLACE TABLE `govuk-bigquery-analytics.banner_intervention.email_one_result` AS
+
+-- All distinct sessions that select the checker via the email
+
+WITH
+  sessions_email_select AS (
+    SELECT DISTINCT
+      CONCAT(fullVisitorId, "-", visitId) AS sessionId
+    FROM `govuk-bigquery-analytics.87773428.ga_sessions_*`
+    CROSS JOIN UNNEST(hits) AS hits
+    WHERE _TABLE_SUFFIX BETWEEN start_date AND end_date
+      AND STARTS_WITH(hits.page.pagePath, "/next-steps-for-your-business?crn")
+  ),
+
+-- All distinct sessions that select the checker via the email that also selects
+-- at least one results link on the results page
+
+  sessions_email_results AS (
+    SELECT DISTINCT
+      CONCAT(fullVisitorId, "-", visitId) AS sessionId
+    FROM `govuk-bigquery-analytics.87773428.ga_sessions_*`
+    CROSS JOIN UNNEST(hits) AS hits
+    WHERE _TABLE_SUFFIX BETWEEN start_date AND end_date
```

Comment:
do we need to check that this click is on a results page?

Reply:
I've popped in the README file some info on this (starts at line 68). For now, no, `SmartAnswerClicked` is the only eventCategory for clicking on a results link. However, this might change. As these analyses are static (i.e. we only are calculating from a specific date range), I think it is fine to just filter on `SmartAnswerClicked`

Reply:
Ahhh cool, such a generic event name but glad to know if can't be triggered on other smart answer results yet :)